blackbox/docs/README.rst

```diff
@@ -0,0 +1,59 @@
+====================
+Documentation README
+====================
+
+Top-Level Structure
+===================
+
+The documentation is structured as follows.
+
++------------------------+----------------------------------------------------+
+| Section Title          |  Description                                       |
++========================+====================================================+
+| Release Notes          | A collection of documents, one document per        |
+|                        | release. Each document should contain upgrade      |
+|                        | information as well as a list of changes.          |
++------------------------+----------------------------------------------------+
+| Installation           | THIS SECTION IS CHANGING SOON                      |
++------------------------+----------------------------------------------------+
+| Configuration          | Documentation for configuration settings, split    |
+|                        | into sections by theme.                            |
++------------------------+----------------------------------------------------+
+| Command Line Interface | Documentation for the ``crate`` command.           |
```

Comment:
you mean crash?

Reply:
then `bin/crate` is not Command Line Interface, it's the script to start the server.

Reply:
@matriv I made this argument before but @joemoe convinced me otherwise. now I don't know what to think lol

joe's argument was that the `crate` command *is* the interface to starting CrateDB

so anyway, this README is mirroring what we have in the docs at present

see: https://crate.io/docs/crate/reference/en/latest/cli.html

I previously suggested to joe that we rename this "Command Line Tool" or maybe we could rename it "Command Line Options" or "Command Line Use" idk

Reply:
I'd go with `Command Line Options`.

Reply:
@joemoe what about "Running CrateDB"

then in this doc I can have a section on the command line tool. and then link through to sections int the guide about deployment, docker, setting up clusters, etc